packages/tools/locales/sr_RS.po

```diff
@@ -4212,17 +4183,16 @@ msgid "To retry decryption of these items. Run `e2ee decrypt --retry-failed-item
 msgstr ""
 
 #: packages/app-cli/app/command-tag.js:14
-#, fuzzy
 msgid ""
 "<tag-command> can be \"add\", \"remove\", \"list\", or \"notetags\" to assign or "
 "remove [tag] from [note], to list notes associated with [tag], or to list tags "
 "associated with [note]. The command `tag list` can be used to list all the tags "
 "(use -l for long option)."
 msgstr ""
-"<tag-command> може бити \"додај\", \"уклони\" или \"листа\" за додељивање или "
-"уклањање [ознаке]  из [белешке], или да излистате белешке повезане са ознаком "
-"[tag]. Команда `tag list` може се користити да излиста све ознаке (користите -l "
-"за дугачки избор)."
+"<tag-command> може бити \"додај\", \"уклони\", \"листа\" или \"ознаке\" за "
+"додељивање или уклањање [tag] из [note], или да излистате белешке повезане са "
```

Comment:
The ones in bracket you can leave translated, because these you will replace with your tag names.

e.g. it's the same for a man page `cp <FILE> <PATH>`. You can't translate `cp` but you can translate FILE and PATH.

Reply:
Yes, I got you. This is for cli :D wasn't looking.

Reply:
Great, thanks for the changes!